apps/debate/models.py

```diff
@@ -73,14 +73,15 @@ def comment_creator_count_minus_three(self):
 
     @cached_property
     def last_three_creators(self):
-        comments = self.comments.all().select_related("creator").order_by("-created")
+        comments = self.comments.filter(is_censored=False, is_removed=False)
+        comments = comments.select_related("creator").order_by("-created")
+
```

Comment:
nice. and just do the filter and select_related in one query. we can have new lines in one queryset for every `.`

Reply:
done